.github/workflows/on-pull-request.yml

```diff
@@ -1,37 +1,42 @@
-name: Run Tests on PR
+name: Run Tests
 
 on:
-    pull_request:
-    workflow_dispatch:
+  workflow_call:
+  workflow_dispatch:
+  pull_request:
 
 env:
-    API_KEY: ${{ secrets.API_KEY }}
-    APP_ID: ${{ secrets.APP_ID }}
-    APP_TOKEN: ${{ secrets.APP_TOKEN }}
-    EXAMPLE_USER_ID: ${{ secrets.EXAMPLE_USER_ID }}
-    PUBLIC_KEY: ${{ secrets.PUBLIC_KEY }}
+  API_KEY: ${{ secrets.API_KEY }}
+  APP_ID: ${{ secrets.APP_ID }}
+  APP_TOKEN: ${{ secrets.APP_TOKEN }}
+  EXAMPLE_USER_ID: ${{ secrets.EXAMPLE_USER_ID }}
+  PUBLIC_KEY: ${{ secrets.PUBLIC_KEY }}
```

Comment:
Is formatting of YAML files in scope of this work too? I think I'd assume this change was caused by prettier, though the adjacent [deploy.yml](https://github.com/passageidentity/passage-node/blob/bae2833f694744a7a34f93d22eac4f83047f733c/.github/workflows/deploy.yml) is still using the old indent.

So this is likely an edge case due to the JS formatter also applying to YAML, but just wanted to verify what to look for here vs. in other SDK repos.

Reply:
the deploy.yml file wasn't touched since it's scoped for a different ticket (with the release CI workflows).

this was a manual change (though based on yaml formatting from the formatters in our other repos).

figured we could add a yaml formatter as a dev container extension since the formatter would likely only run when our internal team makes CI changes.

though for the sake of thoroughness and consistency, i can add it to the on-PR CI if that's more helpful

Reply:
> figured we could add a yaml formatter as a dev container extension since the formatter would likely only run when our internal team makes CI changes.

👍 I think this would likely be easier to apply across the repos than adding it to the PR checks in CI.

A useful follow-up might be to do a pass over any existing YAML files once that's in place. Just thinking of other workflows or actions that might exist that might not necessarily be touched with these tickets, like this [yank workflow](https://github.com/passageidentity/passage-flex-rust/blob/main/.github/workflows/yank.yml) for example. But that seems to be an exception; most repos just have 2 simple workflows.

Reply:
added tickets to run the formatter at the end